CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ### 🔄 Changed
 - Make `LogDetails` fields `public` so they are be accessible. Typical usage is when overriding `process(logDetails:)` when subclassing `BaseLogDestination` [#1650](https://github.com/GetStream/stream-chat-swift/issues/1650)
+- `StreamChatUI` now points towards SwiftyGif version [5.4.1](https://github.com/kirualex/SwiftyGif/releases/tag/5.4.1)) that contains crashfixes.
```

Comment:
This is already added to the changelog 🤔

Reply:
Thanks. I removed that duplicate :)